src-electron/zcl/zcl-loader-silabs.js

```diff
@@ -556,14 +556,15 @@ function prepareCluster(cluster, context, isExtension = false) {
           attribute.$.reportingPolicy
         )
       }
-      let storagePolicy = dbEnum.storageOption.ram
-      if (
-        context.listsUseAttributeAccessInterface &&
+      let storagePolicy = dbEnum.storagePolicy.any
+      if (context.listsUseAttributeAccessInterface && attribute.$.entryType) {
+        storagePolicy = dbEnum.storagePolicy.attributeAccessInterface
+      } else if (
```

Comment:
add `context.listsUseAttributeAccessInterface &&` as per https://github.com/project-chip/zap/pull/1137/files ?

Reply:
Ah my bad. That has to be removed. Ignore me.